src/include/com/amazonaws/kinesis/video/common/Include.h

```diff
@@ -669,6 +677,10 @@ PUBLIC_API STATUS createLwsIotCredentialProviderWithTime(PCHAR, PCHAR, PCHAR, PC
  */
 PUBLIC_API STATUS freeIotCredentialProvider(PAwsCredentialProvider*);
 
+STATUS createLwsEcsCredentialProvider(PCHAR, PCHAR, PAwsCredentialProvider*);

```

Comment:
Comments with descriptions of the params

Reply:
Done